components/data-services/org.wso2.carbon.dataservices.core/src/main/java/org/wso2/carbon/dataservices/core/DBDeployer.java

```diff
@@ -203,6 +204,14 @@ public void deploy(DeploymentFileData deploymentFileData)
             /* save original value */
             serviceActive = service.isActive();
 
+            /* set transports */
+            if(service.getExposedTransports().isEmpty()) {
+				List<String> transports = new ArrayList<String>();
+				transports.add(Constants.TRANSPORT_HTTP);
+				transports.add(Constants.TRANSPORT_HTTPS);
+				service.setExposedTransports(transports);
+			}
```

Comment:
Fixed in https://github.com/wso2/carbon-data/pull/150/commits/c5ba812847dffe3aea4219dcc20290227b3f9682